### PR TITLE
Enable premixing stage1 workflow for phase2

### PR DESCRIPTION
**Pull request by makortel**
This PR enables premixing stage1 workflow for phase2 (ttbar D17 for now) in the default matrix. It does some finishing touches on top of #22375 :
* Add PU definition to the workflow parameters
* Drop DIGI2RAW from the workflow steps
* Customize `PREMIX` event content to include digis
   * Here I'm unsure if it would be better to defined another event content (e.g. `PREMIXDIGI`) and use that

I can't exclude further changes on the stage1 workflow, but this setup turned out to be enough to get work on stage2 workflow started. I think it is in good-enough shape now to be tested regularly in IBs.

Tested in CMSSW_10_1_X_2018-03-05-2300, no changes expected in existing workflows.

@mdhildreth @kpedro88

### CODE REVIEW

**Comment by makortel** [comment]
@cmsbuild, please test workflow 20261.98

File: Configuration/PyReleaseValidation/python/relval_2023.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 #2023 WFs to run in IB (TTbar, TTbar+Timing)
 numWFIB = [20034.0,20034.1,20034.11] #2023D17 w/ special tracking and timing workflows
+numWFIB.extend([20261.98]) # 2023D7 premixing stage1 (NuGun+PU)
```
Comment (kpedro88) [inline Configuration/PyReleaseValidation/python/relval_2023.py]: D17, not D7

Comment (makortel) [reply Configuration/PyReleaseValidation/python/relval_2023.py]: Thanks, fixed.

**Comment by davidlange6** [comment]
please test

**Comment by kpedro88** [comment]
+1

**Comment by makortel** [comment]
Ping

**Comment by prebello** [comment]
+1

**Comment by fabiocos** [comment]
+operations

**Comment by fabiocos** [comment]
+1

**Pull request merged**